octobot/strategy_optimizer/strategy_design_optimizer.py

```diff
@@ -825,7 +832,26 @@ def _get_all_possible_values(self, start, stop, step):
         while current <= d_stop:
             yield return_type(current)
             current += d_step
-
+            
+    def _generate_possible_multiple_options_values(
+        self,
+        possible_values: list,
+        config_element_values,
+        parent_values: list = [],
+    ):
+        for combineable_value in config_element_values:
+            if combineable_value not in parent_values:
+                new_parent_values = parent_values + [combineable_value]
```

Comment:
Would it be better if we have something like this instead?

```python
def _generate_possible_multiple_options_values(
    self,
    possible_values: list,
    config_element_values,
    parent_values: Optional[list] = None,
):
    if parent_values is None:
        parent_values = []
    # rest of the code...
```

Reply:
Using a mutable default argument like `parent_values=[]` can lead to unintended side effects because the same list instance is shared across all calls to the function.

Reply:
Hi @ruidazeng 
ah I see thanks, I changed it

Reply:
Awesome! @Herklos is this something merge-able?